templates/explore/users.tmpl

```diff
@@ -6,6 +6,7 @@
 
 		<div class="ui user list">
 			{{range .Users}}
+				{{if not .HideFromExplorePage}}
```

Comment:
We should filter them when Query but not in the UI. Assume all 10 users on this page chosed hide themselves? What will happen?

Reply:
Oh, it's true. I'll update query then.